railties/lib/rails/generators/test_unit/scaffold/scaffold_generator.rb

```diff
@@ -17,6 +17,10 @@ def create_test_files
         template_file = options.api? ? "api_functional_test.rb" : "functional_test.rb"
         template template_file,
                  File.join("test/controllers", controller_class_path, "#{controller_file_name}_controller_test.rb")
+
+        unless options.api?
+          template "system_test.rb", File.join("test/system", class_path, "#{file_name.pluralize}_test.rb")
+        end
```

Comment:
I think this needs to check `depends_on_system_test?` because some users may want to skip system tests when generating the scaffold.